### ISSUE TITLE
Generated types are ignoring nullable option for arguments

### ISSUE THREAD

**Issue by lagerone** [nenoNaninu/TypedSignalR.Client#159]
When using a nullable argument on a client method, the generated types are missing the nullable specifier.

## Example

```csharp
public interface IChatHub
{
    Task SendMessage(string user, string? message);
}

public interface IChatHubClient
{
    Task ReceiveMessage(string user, string? message);
}

public class ChatHub : Hub<IChatHubClient>, IChatHub
{
    public async Task SendMessage(string user, string? message)
    {
        await Clients.All.ReceiveMessage(user, message);
    }
}

public class Program
{
    public async Task Start()
    {
        HubConnection connection = new HubConnectionBuilder()
            .WithUrl("/hello/world")
            .WithAutomaticReconnect()
            .Build();

        IChatHubClient hubProxy = connection.CreateHubProxy<IChatHubClient>();

        await connection.StartAsync();
    }
}
```

This generates the following `ReceiveMessage` type

![image](https://github.com/nenoNaninu/TypedSignalR.Client/assets/1382720/0fe0fa99-d295-4779-ad92-61bb65e53586)

I would expect the `message` argument to be nullable here.

Are nullable arguments not supported by TypedSignalR? Or is this a bug?

**Comment by nenoNaninu** [comment]
Thanks for reporting the issue. It is just a warning and does not affect execution because nullable annotations for reference types are only used for flow analysis.

However, since this warning is unsuitable, I fixed this issue and released `v3.4.2`. Try it!

**Comment by lagerone** [comment]
Thanks for the quick response and fix! The warning is gone 👏 

The generated types still looks "wrong" to me though (missing nullable) but since the studio still understands the code correctly I guess it's nothing to worry about.
